packages/app/src/cli/commands/app/logs.ts

```diff
@@ -37,7 +37,7 @@ export default class Logs extends Command {
     'no-tunnel': Dev.flags['no-tunnel'],
     'graphiql-port': Dev.flags['graphiql-port'],
     'graphiql-key': Dev.flags['graphiql-key'],
-    'dev-preview': Dev.flags['dev-preview'],
+    stable: Dev.flags.stable,
```

Comment:
I asked how this works [here](https://shopify.slack.com/archives/C030LHFCA5T/p1726092713391289).

Reply:
I've answered this in the thread below :)